static/scripts/main.js

```diff
@@ -6,23 +6,39 @@ window.onload = function () {
     if(hash) {
       hash.scrollIntoView()
     }
+    $('[data-md-sticky-header]').headroom({
+      offset: UnpinOffset,
+      tolerance: 0
+    });
 }
 
-var CurrentScroll = 0;
+var SearchbarTop = 61;
+var UnpinOffset = 400;
+var PrevScrollTop = 0;
 
 $(window).scroll(function() {
-  var NextScroll = $(this).scrollTop();
-  if (NextScroll < CurrentScroll && NextScroll > 67){
-     //upscroll
-     $('[data-md-sticky-header]').addClass('sticky');
+  function handleScrollUp() {
+    if (ScrollTop > SearchbarTop && ScrollTop <= UnpinOffset)
```

Comment:
yeah, even single-statement blocks should be in braces

this code (I mean the entire source, not this PR specifically) actually needs linting, for example, variables should start from lower-case letter